ext/standard/file.h

```diff
@@ -48,7 +48,9 @@ PHPAPI void php_flock_common(php_stream *stream, zend_long operation, uint32_t o
 	zval *wouldblock, zval *return_value);
 
 #define PHP_CSV_NO_ESCAPE EOF
-PHPAPI void php_fgetcsv(php_stream *stream, char delimiter, char enclosure, int escape_char, size_t buf_len, char *buf, zval *return_value);
+#define BC_EMPTY_CSV_LINE_ARRAY(values) { (values) = zend_new_array(1); zval _______z_tmp; \
+	ZVAL_NULL(&_______z_tmp); zend_hash_next_index_insert((values), &_______z_tmp); }
```

Comment:
It seems to me an inline function would be preferable.

Reply:
ack